Reference/Searching/Examine/index-v7.md

```diff
@@ -0,0 +1,37 @@
+---
+versionFrom: 7.0.0
+---
+
+# Examine
+
+_Examine uses Lucene as its search and index engine. Searching using Examine with Lucene can be very powerful and fast and it can also be really simple to setup and use._
+
+## What is Examine?
+
+Examine allows you to index and search data quickly and easily. It is a library that sits on top of Lucene, a very high performance search engine library written in Java. Examine is built on top of a .NET-implementation of Lucene to provide very efficient APIs to make searching and indexing as straight forward as possible. Umbraco provides a further layer on top, UmbracoExamine, that opens up the Umbraco-specific APIs for indexing and searching content and media out of the box.
+
+Examine is provider based so is very extensible and allows you to configure as many indexes as you like and each may be configured individually. The backoffice search in Umbraco also uses this same search engine, so you can trust that you're in good hands.
```

Comment:
Should be:

> Examine is provider based so it is extensible and allows you to configure your own custom indexes if required. The backoffice search in Umbraco also uses this same search engine, so you can trust that you're in good hands.

No need to use words like 'very' all of the time, we don't need to try to impress people reading the docs :) ... but i suspect this was copy/pasted from before. We should also not encourage people to create 'as many indexes as they want' because we don't want people to go creating a ton of indexes since each index added adds overhead equivalent to another index.